lib/vcard/dirinfo.rb

```diff
@@ -242,11 +242,14 @@ def delete(field)
       self
     end
 
+    LF= "\n"
+
     # The string encoding of the DirectoryInfo. See Field#encode for information
-    # about the width parameter.
-    def encode(width=75)
+    # about the parameters.
+    def encode(width = 75, nl: LF)
       unless @string
-        @string = @fields.collect { |f| f.encode(width) }.join ""
+        options = [width, nl: nl]
+        @string = @fields.collect { |f| f.encode(*options) }.join
```

Comment:
How about removing the `options` variable and just do:

```
@string = @fields.collect { |f| f.encode(width, nl: nl) }.join
```

?

Reply:
Done!